module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,12 @@
+#include "vowel.hpp"
+#include <algorithm>
+
+
+void removeVowels(std::vector<std::string>& vec){
+    std::vector<char> vowelVector {'a', 'A', 'e', 'E', 'i', 'I', 'o', 'O', 'u', 'U', 'y', 'Y'};
```

Comment:
I think you can make it constrexpr and I would consider moving it to the header.